.vscode/tasks.json

```diff
@@ -20,13 +20,20 @@
     {
       "taskName": "storybook",
       "args": [
-        "run",
         "storybook"
       ]
     },
     {
+      "taskName": "TypeScript: Run type-checker",
+      "command": "tsc",
+      "isShellCommand": true,
+      "args": ["-w", "-p", ".", "--noEmit"],
+      "showOutput": "silent",
+      "isBackground": true,
+      "problemMatcher": "$tsc-watch"
+    },
```

Comment:
I haven’t found a way to auto-start this when you open the project, does either of you know?